lib/delay.rb

```diff
@@ -151,7 +157,7 @@ module Validator
     module ClassMethods
       def validates_delay_format_of(*attr_names)
         attr_names.each do |attr_name|
-          validates attr_name, delay: true
+          validates attr_name, delay: { message: proc { I18n.translate('activerecord.errors.models.purchase.payment_delay_custom_validation_message', values: (TRANSLATIONS[Preference[:language].to_sym]&.keys.join(', ') || TRANSLATIONS[:eng].keys.join(', ') )) } }
```

Comment:
This is way too long and hard to read.
I think a good idea could be either to set the proc into a constant in the class:
```suggestion
BAD_DELAY_MESSAGE_PROC = proc do 
  # ...
end

  # ...

  validates attr_name, delay: { message: BAD_DELAY_MESSAGE_PROC }
```
or into a class method:
```suggestion
  def self.bad_delay_message(*args)
     # ...
  end

  # ...
  validates attr_name, delay: { message: method(:bad_delay_message) }
```
Either way once extracted it should be refactored it into a multiline expression using properly named variables.